cognite_toolkit/_cdf_tk/load/_resource_loaders.py

```diff
@@ -1899,3 +1909,114 @@ def count(self, ids: SequenceNotStr[NodeId]) -> int:
+        upserts = [items] if isinstance(items, WorkflowUpsert) else items
+        return WorkflowList([self.client.workflows.upsert(upsert) for upsert in upserts])
+
+    def create(self, items: WorkflowUpsert | WorkflowUpsertList) -> WorkflowList:
+        return self._upsert(items)
+
+    def update(self, items: WorkflowUpsertList) -> WorkflowList:
+        return self._upsert(items)
+
+
+@final
+class WorkflowVersionLoader(
+    ResourceLoader[
+        WorkflowVersionId, WorkflowVersionUpsert, WorkflowVersion, WorkflowVersionUpsertList, WorkflowVersionList
+    ]
+):
+    api_name = "workflows.versions"
+    folder_name = "workflows"
+    filename_pattern = r"^.*\.?(WorkflowVersion)$"
+    resource_cls = WorkflowVersion
+    resource_write_cls = WorkflowVersionUpsert
+    list_cls = WorkflowVersionList
+    list_write_cls = WorkflowVersionUpsertList
+    dependencies = frozenset({WorkflowLoader})
+
+    @classmethod
+    def get_required_capability(cls, items: WorkflowVersionUpsertList) -> Capability:
+        return WorkflowOrchestrationAcl(
+            [WorkflowOrchestrationAcl.Action.Read, WorkflowOrchestrationAcl.Action.Write],
+            WorkflowOrchestrationAcl.Scope.All(),
+        )
+
+    @classmethod
+    def get_id(cls, item: WorkflowVersion | WorkflowVersionUpsert) -> WorkflowVersionId:
+        return item.as_id()
+
+    def load_resource(
+        self, filepath: Path, ToolGlobals: CDFToolConfig, skip_validation: bool
+    ) -> WorkflowVersionUpsertList:
+        resource = load_yaml_inject_variables(filepath, {})
+
+        workflowversions = [resource] if isinstance(resource, dict) else resource
+        return WorkflowVersionUpsertList.load(workflowversions)
+
+    def retrieve(self, ids: SequenceNotStr[WorkflowVersionId]) -> WorkflowVersionList:
+        return self.client.workflows.versions.list([id for id in ids])
+
+    def _upsert(self, items: WorkflowVersionUpsertList) -> WorkflowVersionList:
+        return WorkflowVersionList([self.client.workflows.versions.upsert(item) for item in items])
+
+    def create(self, items: WorkflowVersionUpsertList) -> WorkflowVersionList:
+        upserted = []
+        for item in items:
+            upserted.append(self.client.workflows.versions.upsert(item))
+        return WorkflowVersionList(upserted)
+
+    def update(self, items: WorkflowVersionUpsertList) -> WorkflowVersionList:
+        updated = []
+        for item in items:
+            updated.append(self.client.workflows.versions.upsert(item))
+        return WorkflowVersionList(updated)
+
+    def delete(self, ids: SequenceNotStr[WorkflowVersionId]) -> int:
+        successes = 0
+        for id in ids:
+            try:
+                self.client.workflows.versions.delete(id)
+                successes += 1
+            except CogniteNotFoundError:
+                print(f"  [bold yellow]WARNING:[/] WorkflowVersion {id} does not exist, skipping delete.")
```

Comment:
You are only doing a try on the `.delete` call, so lets move the incrementation to `else` block as that is only executed on success. 
```suggestion
        for id in ids:
            try:
                self.client.workflows.versions.delete(id)
            except CogniteNotFoundError:
                print(f"  [bold yellow]WARNING:[/] WorkflowVersion {id} does not exist, skipping delete.")
            else:
                successes += 1
```